routes/search/operations.js

```diff
@@ -0,0 +1,38 @@
+module.exports = [
+  {
+    name: 'docs',
+    displayName: 'Documentations',
```

Comment:
nit: Documentation is in itself a plural I believe, so this should be `Documentation`